src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -2179,6 +2186,31 @@ private void RestoreInternalDebugMode()
 
         #region Debugger Overrides
 
+        /// <summary>
+        /// Adds the provided set of breakpoints to the debugger.
+        /// </summary>
+        /// <param name="breakpoints"></param>
+        public override void SetBreakpoints(IEnumerable<Breakpoint> breakpoints)
```

Comment:
Can you put this method together with `SetCommandBreakpoint`, `SetLineBreakpoint` and `SetVariableBreakpoint`?